2 - RTB/Documentazione interna/Verbali/Esterni/23-11-15/23-11-15.typ

```diff
@@ -0,0 +1,116 @@
+=== Utente
+L'utente tipico è stato individuato nella figura di un amministratore e/o impiegato lato back-office, dotati dunque di massimo controllo e accesso all'applicativo. \
+Le principali azioni che tali utenti svolgeranno sono:
+  - interrogazione del magazzino;
+  - richiesta di spostamento di un prodotto;
+  - zoom e spostamento della camera;
+  - interrogazione in merito alla presenza o meno di prodotti.
+
+=== User interface & user experience
+Durante lo svolgimento del meeting si è presentata un'idea generale di possibile user interface, delineando in particolar modo alcune caratteristiche desiderabili, quali:
+- rendere ben visibili le operazioni possibili (distinzione evidente tra elementi selezionabili e non);
+- evitare eccessive animazioni o effetti _lampeggianti_;
+- evidenziare in modo visibile gli elementi ricercati (magari riducendo l'opacità degli altri elementi);
+- spostare eventualmente la telecamera sull'oggetto ricercato (se singolo);
+- essenziale che la ricerca non modifichi la struttura visualizzata del magazzino.
+
+=== Differenze tra applicativo desktop e mobile
+Durante la riunione è emerso che vi saranno delle differenze tra la versione desktop e la versione mobile: l'accesso completo alla funzionalità (esempio richiesta di spostamento di prodotti) sarà fornito per la versione desktop, mentre la versione mobile presenterà delle funzioni limitate (magari solo visualizzazione e ricerca). In ogni caso rimane ancora definire nel dettaglio le differenze.
+
+=== Funzionalità desiderate
+Da una prima analisi sono emerse le principali funzionalità desiderate, quali:
+- creazione dell'ambiente 3D del magazzino, articolata in:
+  - determinazione dei parametri dimensionali del magazzino;
+  - determinazione dei bin;
+  - creazione e posizionamento degli scaffali;
+- caricamento dei dati da un database per la popolazione del magazzino;
+- interrogazione del magazzino mediante ricerce per ID, nome, o caratteristiche similari dei prodotti;
+- individuazione visibile (mediante cambio colore o spostamento della camera) di prodotti ricercati;
+- invio di notifiche mediante la chiamata a API per lo spostamento di prodotti nel magazzino.
+
+Durante l'analisi del processo di creazione si è fatta chiarezza relativamente al rapporto tra creazione dell'ambiente 3D e dati presenti nel database:
+sostanzialmente, il primo passaggio è la creazione dell'ambiente 3D. Il database non impone vincoli strutturali, ma bensì è pensato per interfacciarsi con la struttura determinata in fase di creazione. 
+
+=== API
+Per l'invio delle notifiche di richiesta di spostamento, l'applicativo si interfaccerà con delle API. L'implementazione è emerso non essere oggetto del capitolato, e per tale motivazione possiamo porcedere ad una simulazione del comportamento delle stesse con assunzioni riguardo i parametri e le risposte. Durante il meeting, si è individuato un possibile standard. Esempio per lo spostamento di un elemento da un bin ad un altro:
```

Comment:
"si è individuato un possibile standard". Quale?

Reply:
Inizialmente mi riferivo all'esempio con "invio le coordinate di partenza, di arrivo e il prodotto" e in output ottengo "risposta cod 200 con json contente l'esito", ma capisco che è fuorviante. Meglio rimuoverlo.